vietnamese/TansStory/mystory.md

```diff
@@ -0,0 +1,2 @@
+Minh di dinh cu o My cung duoc 10 nam roi. Minh moi hoc ra truong.
+Minh dang kiem [viec lam](Tanstory/mystory.md)ma van khong co. Nhung minh se khong nan ma bo cuoc. Minh se co gang nhieu hon nua.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.